tests/descriptors/analysis_file/f0c8ac7c-1c4a-5c9c-9286-c8b16208f9ae_2021-01-01T00:00:00.000000Z.json

```diff
@@ -0,0 +1,14 @@
+{
+    "content_type": "application/gzip; dcp-type=data",
+    "crc32c": "4dab8938",
+    "describedBy": "https://schema.staging.data.humancellatlas.org/system/2.0.0/file_descriptor",
```

Comment:
The value of `describedBy` here fails against the `https://schema.staging.data.humancellatlas.org/system/2.0.0/file_descriptor` schema because the regex `schema(\\.[a-z]+)?\\.humancellatlas` does not allow for extra `.` (dot) characters between `schema.` and `.humancellatlas`.

Other schemas like `https://schema.staging.data.humancellatlas.org/type/biomaterial/13.3.0/cell_suspension` do not have this issue due to a different regex approach `schema.(.*?)humancellatlas`

Reply:
Thanks for catching this @danielsotirhos! 

This schema was created for the DCP2 so I think this is the first time we ever created it in ingest staging, so we overlooked that, apologies!

We will update this! We are discussing the approaches and we have reduced it to 2 questions I will like to ask:
- Do we want to treat this as a bugfix (No version update, re-publish schema) or do we want to add it to go through the usual process and update the version? (We'll need to regenerate test data if that's the case)
- Do we want to include it in the existing schema release PR, or do we want to just move it through dev, staging and master as a hotfix?
@hannes-ucsc @HumanCellAtlas/wranglers @aaclan-ebi

Reply:
I agree. We should just add the schema regex fix as part of the metadata-schema PR https://github.com/HumanCellAtlas/metadata-schema/pull/1371/files and regenerate the test data.

Reply:
@danielsotirhos please verify resolution.

Reply:
Confirmed the issue with the `file_descriptor` schema has been resolved, and that all files in this canned staging area as of commit 2c7fbd0e0b7804abeb21ddb0ae89724950218257 pass JSON validation against their schema.